README.md

```diff
@@ -321,6 +321,10 @@ In most case you want to use the `inline` cache exporter.
 However, note that the `inline` cache exporter only supports `min` cache mode. 
 To enable `max` cache mode, push the image and the cache separately by using `registry` cache exporter.
 
+`inline` and `registry` exporters both store the cache in the registry. For importing the cache, `type=registry` is sufficient for both, as specifying the cache format is not necessary.
+
+Cache keys are defined by source uniqueness of your build dependencies. If your build uses only specific files and they are the same in multiple branches then cache will be reused. In `gha` there is additional namespacing by branch scope for security.
```

Comment:
This section is a bit weird here as there is nothing special about what is cached with remote cache. "multiple branches" also confusing here. Better to drop and explain Github cache separately.

Reply:
https://github.com/moby/buildkit/pull/2489/commits/3d68f3bbf940486c6115f947fea77df3c93d0622